src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -403,6 +403,10 @@ public static partial class Marshal
         public static string PtrToStringUni(System.IntPtr ptr) { return default(string); }
         [System.Security.SecurityCriticalAttribute]
         public static string PtrToStringUni(System.IntPtr ptr, int len) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringUTF8(System.IntPtr ptr) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringUTF8(System.IntPtr ptr, int byteLen) { return default(string); }        
```

Comment:
All the other `PtrToStringXxx` methods have a parameter named `len`, not `byteLen`. Is the naming divergence a deliberate choice?